packages/apps/explorer/src/components/CompactTable/utils/formatXChainTX.tsx

```diff
@@ -0,0 +1,14 @@
+import { MonoAirlineStops } from '@kadena/kode-icons/system';
+import type { FC } from 'react';
+import React from 'react';
+
+interface IProps {
+  value: string;
+}
+
+export const formatXChainTX = (): FC<IProps> => {
```

Comment:
```suggestion
export const formatMultiStepTx = (): FC<IProps> => {
```
Maybe change the name since we now look only to `continuation`